docs/cli/product-json-to-csv.md

```diff
@@ -0,0 +1,161 @@
+# Product JSON to CSV Parser
+
+Convert [commercetools Products](http://dev.commercetools.com/http-api-projects-discountCodes.html) CSV data to JSON. See example below for CSV format, sample response and usage.
```

Comment:
Hmmmm isn't it the other way around? JSON data to CSV? I guess we can just remove this sentence in favor of the one below.